File: CS/WindowsApplication3/ItemList.cs
using System.ComponentModel;

namespace WindowsApplication3
{
    public static class ItemList
    {
        public static BindingList<Item> GetList()
        {
            var list = new BindingList<Item>();
            for (int i = 0; i < 50; i++)
                list.Add(new Item() { Column1 = "Test1" + i, Column2 = "Test2" + i, Column3 = "Test3" + i, Column4 = "Test4" + i, Column5 = "Test5" + i, Column6 = "Test6" + i, Column7 = "Test7" + i, Column8 = "Test8" + i, Column9 = "Test9" + i, Column10 = "Test10" + i, Column11 = "Test11" + i, Column12 = "Test12" + i, Column13 = "Test13" + i, Column14 = "Test14" + i, Column15 = "Test15" + i, Column16 = "Test16" + i });
            return list;
        }
    }
    public class Item
    {
        public string Column1 { get; set; }
        public string Column2 { get; set; }
        public string Column3 { get; set; }
        public string Column4 { get; set; }
        public string Column5 { get; set; }
        public string Column6 { get; set; }
        public string Column7 { get; set; }
        public string Column8 { get; set; }
        public string Column9 { get; set; }
        public string Column10 { get; set; }
        public string Column11 { get; set; }
        public string Column12 { get; set; }
        public string Column13 { get; set; }
        public string Column14 { get; set; }
        public string Column15 { get; set; }
        public string Column16 { get; set; }
    }
}

File: VB/WindowsApplication3/ItemList.vb
Imports System.ComponentModel

Namespace WindowsApplication3
    Public NotInheritable Class ItemList

        Private Sub New()
        End Sub

        Public Shared Function GetList() As BindingList(Of Item)
            Dim list = New BindingList(Of Item)()
            For i As Integer = 0 To 49
                list.Add(New Item() With { _
                    .Column1 = "Test1" & i, _
                    .Column2 = "Test2" & i, _
                    .Column3 = "Test3" & i, _
                    .Column4 = "Test4" & i, _
                    .Column5 = "Test5" & i, _
                    .Column6 = "Test6" & i, _
                    .Column7 = "Test7" & i, _
                    .Column8 = "Test8" & i, _
                    .Column9 = "Test9" & i, _
                    .Column10 = "Test10" & i, _
                    .Column11 = "Test11" & i, _
                    .Column12 = "Test12" & i, _
                    .Column13 = "Test13" & i, _
                    .Column14 = "Test14" & i, _
                    .Column15 = "Test15" & i, _
                    .Column16 = "Test16" & i _
                })
            Next i
            Return list
        End Function
    End Class
    Public Class Item
        Public Property Column1() As String
        Public Property Column2() As String
        Public Property Column3() As String
        Public Property Column4() As String
        Public Property Column5() As String
        Public Property Column6() As String
        Public Property Column7() As String
        Public Property Column8() As String
        Public Property Column9() As String
        Public Property Column10() As String
        Public Property Column11() As String
        Public Property Column12() As String
        Public Property Column13() As String
        Public Property Column14() As String
        Public Property Column15() As String
        Public Property Column16() As String
    End Class
End Namespace
